bciers/apps/administration/app/components/operators/OperatorDataGrid.tsx

```diff
@@ -3,11 +3,19 @@
 import { useMemo, useState } from "react";
 import DataGrid from "@bciers/components/datagrid/DataGrid";
 import HeaderSearchCell from "@bciers/components/datagrid/cells/HeaderSearchCell";
-import OperatorsActionCell from "@bciers/components/datagrid/cells/OperatorsActionCell";
 import fetchOperatorsPageData from "./fetchOperatorsPageData";
 import { OperatorRow } from "./types";
 import operatorColumns from "../datagrid/models/operatorColumns";
 import operatorGroupColumns from "../datagrid/models/operatorGroupColumns";
+import ActionCellFactory from "@bciers/components/datagrid/cells/ActionCellFactory";
+import { GridRenderCellParams } from "@mui/x-data-grid";
+
+const OperatorsActionCell = ActionCellFactory({
```

Comment:
Could add an assertion in the `OperatorDataGrid` test to make sure the text/link in this column is correct